deploy-operator.sh

```diff
@@ -0,0 +1,124 @@
+set -x
+
+# Create secret (must be named cert.pem)
+cp ../certs/registry.pem ../certs/cert.pem
+oc create secret generic foo-cert-sec --from-file=../certs/cert.pem
+
+# Remove the docker registry 
+docker rm -f registry
+
+# Create the docker registry
+docker run -d \
+  --restart=always \
+  --name registry \
+  -v $(pwd)/../certs:/certs:Z \
+  -e REGISTRY_HTTP_ADDR=0.0.0.0:443 \
+  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/registry.crt \
+  -e REGISTRY_HTTP_TLS_KEY=/certs/registry.key \
+  -p 443:443 \
+  registry:2
+
+# Install operator sdk
+## Configure platform
+export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+export OS=$(uname | awk '{print tolower($0)}')
+
+## Download executable
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.11.0
+curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+
+## Download the auth key
+gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E
+
+curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt
+curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt.asc
+gpg -u "Operator SDK (release) <cncf-operator-sdk@cncf.io>" --verify checksums.txt.asc
+
+OUT=$(grep operator-sdk_${OS}_${ARCH} checksums.txt | sha256sum -c -)
+echo $OUT
+if [ "$OUT"="operator-sdk_linux_amd64: OK" ];
+then
+  chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+  echo "operator-sdk configured"
+else 
+  echo "Error: Checksum mismatch, quitting"
+  exit 1
+fi
+
+# create the test operator
+mkdir nginx-operator
+cd nginx-operator
+operator-sdk init --domain cnftest-local.redhat.com --plugins helm
+ 
+# Create a simple nginx API using Helm’s built-in chart boilerplate (from helm create):
+operator-sdk create api --group cnftest --version v0 --kind Nginx
+
+# Logging registry before using
+docker logs registry 
+
+# Build and push the operator image:
+make docker-build docker-push IMG="cnftest-local.redhat.com/nginx-operator:v0.0.1"
+
+# Install OLM
+operator-sdk olm install
+
+# Configure the operator manifest
+mkdir -p config/manifests/bases
+
+# Create configration
+cat > config/manifests/bases/nginx-operator.clusterserviceversion.yaml <<-'EOF'
```

Comment:
I think it's better to have this in separate file.

Reply:
This file is not really a important configuration that we have to track, more a temporary file to avoid filling the values from cli